README.md

```diff
@@ -54,6 +71,17 @@ sap_hana_ha_pacemaker_hacluster_node1_fqdn: hana-node1.test.local
 sap_hana_ha_pacemaker_hacluster_node2_fqdn: hana-node2.test.local
 sap_hana_ha_pacemaker_hacluster_node1_ip: 192.168.47.21
 sap_hana_ha_pacemaker_hacluster_node2_ip: 192.168.47.22
+sap_hana_ha_pacemaker_fencing_device_ip: 192.168.47.11
```

Comment:
It might be worth to also mention fence_aws as well, which is the fencing agent that hast to be used on AWS, see https://access.redhat.com/articles/3354781#configure-fencing-9